src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private static final double PI = 3.14;
+    private final int radius;
+
+    public Circle(int radius, String color) {
+        this.radius = radius;
+        setColor(color);
+        obtainTheArea();
+    }
+
+    public void obtainTheArea() {
+        setArea(PI * radius * radius);
+    }
```

Comment:
Method 'obtainTheArea' should be private if it's only used within the class. Also, consider renaming it to reflect that it's calculating and setting the area, not just obtaining it. A name like 'calculateAndSetArea' might be more descriptive.